daprdocs/content/en/reference/components-reference/supported-state-stores/setup-etcd.md

```diff
@@ -0,0 +1,86 @@
+---
+type: docs
+title: "Etcd"
+linkTitle: "Etcd"
+description: Detailed information on the Etcd state store component
+aliases:
+- "/operations/components/setup-state-store/supported-state-stores/setup-etcd/"
+---
+
+## Component format
+
+To setup Etcd state store create a component of type `state.etcd`. See [this guide]({{< ref "howto-get-save-state.md#step-1-setup-a-state-store" >}}) on how to create and apply a state store configuration.
+
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: state.etcd
+  version: v1
+  metadata:
+  - name: endpoints
+    value: <CONNECTION STRING> # Required. Example: 192.168.0.1:2379,192.168.0.2:2379,192.168.0.3:2379
+  - name: keyPrefixPath
+    value: <KEY PREFIX STRING> # Optional. default: "". Example: "dapr"
+  - name: tlsEnable
+    value: <ENABLE TLS> # Optional. Example: "false"
+  - name: ca
+    value: <CA> # Optional.
+  - name: cert
+    value: <CERT> # Optional.
+  - name: key
+    value: <KEY> # Optional.
+```
+
+{{% alert title="Warning" color="warning" %}}
+The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
+{{% /alert %}}
+
+## Spec metadata fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| endpoints          | Y        | Connection string to Etcd server | `"192.168.0.1:2379,192.168.0.2:2379,192.168.0.3:2379"`
+| keyPrefixPath      | N        | Key prefix path in Etcd. Default is `""` | `"dapr"`
+| tlsEnable          | N        | Whether to enable tls  | `"false"`
+| ca                 | N        | Contents of Etcd server CA file | `"ca value"`
```

Comment:
@taenyang - For the `ca`, `cert` and `key` examples we have found that either point to documentation or providing more real world examples are better here. The goal here is clarity if there are any specific on how these value should be provided i.e. the Format of the strings. Is there anything special needed?

Reply:
Add reference to use component-secrets and real example values for ca, cert and key.